scripts/archived_scripts/examples_using_data_feeds/amm_data_feed_example.py

```diff
@@ -10,8 +10,8 @@
 
 
 class AMMDataFeedExample(ScriptStrategyBase):
-    amm_data_feed_uniswap = AmmGatewayDataFeed(
-        connector_chain_network="uniswap_polygon_mainnet",
+    amm_data_feed_amm = AmmGatewayDataFeed(
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed

Reply:
same for all of this filed amm_data_feed_example